software/salmon/quant/main.nf

```diff
@@ -41,11 +41,16 @@ process SALMON_QUANT {
         input_reads = "-a $reads"
     }
 
-    def strandedness =  ''
-    if (lib_type) {
+    def strandedness_opts = [
+        'A', 'U', 'SF', 'SR',
+        'IS', 'IU' , 'ISF', 'ISR',
+        'OS', 'OU' , 'OSF', 'OSR',
+        'MS', 'MU' , 'MSF', 'MSR' 
+    ]
+    def strandedness =  'A'
+    if (strandedness_opts.contains(lib_type)) {
         strandedness = "${lib_type}"
-    }
-    else {
+    } else {
```

Comment:
Maybe through a warning if `lib_type` is passed but thas not match any value in `strandedness_opts`

Reply:
Yep, can try to do that 👍🏽

Reply:
Done https://github.com/nf-core/modules/pull/536/commits/b74a93fb9bb09ff439c017cd73d3d819c30bfd94

Reply:
Awesome

Reply:
I tested and works perfectly :+1: